From f7f26baeca210a1b0b5bf71e00f47c9dc12dca3c Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Fri, 5 Dec 2014 12:12:21 -0800
Subject: [PATCH] tests: add rockchip to modetest, kmstest and vbltest

Add "rockchip" to the static lists of driver names in these tests.

Signed-off-by: Daniel Kurtz <djkurtz@chromium.org>
---
 tests/kmstest/main.c      | 1 +
 tests/modetest/modetest.c | 2 +-
 tests/vbltest/vbltest.c   | 2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/tests/kmstest/main.c b/tests/kmstest/main.c
index 449d75f..3f9a292 100644
--- a/tests/kmstest/main.c
+++ b/tests/kmstest/main.c
@@ -62,6 +62,7 @@ char *drivers[] = {
 	"nouveau",
 	"vmwgfx",
 	"exynos",
+	"rockchip",
 	NULL
 };
 
diff --git a/tests/modetest/modetest.c b/tests/modetest/modetest.c
index 7d436b5..a0c3ba9 100644
--- a/tests/modetest/modetest.c
+++ b/tests/modetest/modetest.c
@@ -1435,7 +1435,7 @@ int main(int argc, char **argv)
 	int drop_master = 0;
 	int test_vsync = 0;
 	int test_cursor = 0;
-	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "exynos", "tilcdc", "msm" };
+	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "exynos", "tilcdc", "msm", "rockchip" };
 	char *device = NULL;
 	char *module = NULL;
 	unsigned int i;
diff --git a/tests/vbltest/vbltest.c b/tests/vbltest/vbltest.c
index 2a09d28..96f97f6 100644
--- a/tests/vbltest/vbltest.c
+++ b/tests/vbltest/vbltest.c
@@ -103,7 +103,7 @@ static void usage(char *name)
 int main(int argc, char **argv)
 {
 	int i, c, fd, ret;
-	char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "exynos", "omapdrm", "tilcdc", "msm" };
+	char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "exynos", "omapdrm", "tilcdc", "msm", "rockchip" };
 	drmVBlank vbl;
 	drmEventContext evctx;
 	struct vbl_info handler_info;
-- 
2.2.0.rc0.207.ga3a616c

